<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1403</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1391</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_3{border-image: url(:/back/image/mybackgroundimage.jpg);}
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{border-image: url(:/back/image/left.png);}</string>
    </property>
    <widget class="QPushButton" name="WestLeft">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG.jpg);</string>
     </property>
     <property name="text">
      <string>东1测点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WestSouth">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG.jpg);</string>
     </property>
     <property name="text">
      <string>东2吊杆</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EastSouth">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG.jpg);</string>
     </property>
     <property name="text">
      <string>南4吊杆</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EastNorth">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>230</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG.jpg);</string>
     </property>
     <property name="text">
      <string>南3测点</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2{border-image: url(:/back/image/right.png);}</string>
    </property>
    <widget class="QPushButton" name="WestLeft_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG1.jpg);</string>
     </property>
     <property name="text">
      <string>西5测点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="WestSouth_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>320</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG1.jpg);</string>
     </property>
     <property name="text">
      <string>西6吊杆</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EastNorth_1">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG1.jpg);</string>
     </property>
     <property name="text">
      <string>北7测点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EastSouth_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;
background-image: url(:/back/image/BG1.jpg);</string>
     </property>
     <property name="text">
      <string>北8吊杆</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>841</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>挂篮实物图</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
